### PR TITLE
MIGENG-687 Remove reevaluate for shared disks

### PR DESCRIPTION
**Pull request by m-brophy**
https://issues.redhat.com/browse/MIGENG-687?focusedCommentId=15505116&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15505116

### CODE REVIEW

File: src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReevaluateTest.java [hunk 1]
```diff
@@ -26,7 +26,8 @@
 import static org.junit.Assert.*;
 
 @RunWith(Parameterized.class)
```
Comment (carlosthe19916) [inline src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReevaluateTest.java]: I suggest to delete this test class or just add the `@Ignore` tag on top of the class. It is just a suggestion, what do you think @m-brophy

**Review by carlosthe19916 — CHANGES_REQUESTED** [review]
@m-brophy I added just one comment in this PR. Besides of the comment I made I also think:
- Could we just delete the file [AgendaFocusForReevaluate.drl](https://github.com/project-xavier/xavier-analytics/blob/master/src/main/resources/org/jboss/xavier/analytics/rules/agendafocus/reevaluate/AgendaFocusForReevaluate.drl) since it is not used anymore, just to not have code that is never executed. What do you think?
- My understanding is that `"Shared Disk"` will be completely replaced by "Shared VMDK" so we can safely delete this line [SHARED_DISK_FLAG_NAME = "Shared Disk"](https://github.com/project-xavier/xavier-analytics/blob/master/src/main/java/org/jboss/xavier/analytics/pojo/output/workload/inventory/WorkloadInventoryReportModel.java#L22) ; if we delete this line then the test will fail (which I think is what I expect) and some fixes to the tests will be needed. Maybe I'm missing something @m-brophy , what do you think?

File: src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReevaluateTest.java (hunk 1 above)
Comment (m-brophy) [reply src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReevaluateTest.java]: Agreed, I've deleted the test class,the reevaluate agenda, the constant declaration and a couple of other classes

**Review by carlosthe19916 — APPROVED** [review]
Thanks @m-brophy for taking care of this PR. I tested this PR in CI and everything worked as expected.

**Pull request merged**